docs/API/metrics_api.md

```diff
@@ -1,19 +1,10 @@
-#<a name="top"></a>Metrics API
+## Metrics API
 
--   [Introduction](#introduction)
```

Comment:
Please don't remove ToC in markdown. We don't want to ruin navigability for people that browse documentation in markdown and doesn't use RTD. In theory, the ToC don't appear in RTD so the document in RTD would be also ok.

Reply:
Also the readthedocs CSS has already been altered to suppress the ToC from double displaying on ReadTheDocs anyway, so the readability is not negatively impacted by leaving it in.

Reply:
Fixed in 4b6bedac6c54ebf6fd518840f1d61520f7b528aa